src/policy/rbf.h

```diff
@@ -10,13 +10,22 @@
 enum class RBFTransactionState {
     UNKNOWN,
     REPLACEABLE_BIP125,
-    FINAL
+    FINAL,
 };
 
-// Determine whether an in-mempool transaction is signaling opt-in to RBF
-// according to BIP 125
-// This involves checking sequence numbers of the transaction, as well
-// as the sequence numbers of all in-mempool ancestors.
+/**
+ * Determine whether an unconfirmed transaction is signaling opt-in to RBF
+ * according to BIP 125
+ * This involves checking sequence numbers of the transaction, as well
+ * as the sequence numbers of all in-mempool ancestors.
+ *
+ * @param tx   The unconfirmed transaction
+ * @param pool The mempool, which may contain the tx
+ *
+ * @return     The rbf state. May be UNKNOWN for unconfirmed transactions not
+ *             in the mempool. May be REPLACEABLE_BIP125 for transactions that
+ *             conflict with an in-chain transaction.
```

Comment:
This comment lacks `FINAL` value description: https://github.com/bitcoin/bitcoin/blob/fa9124d93527fd5ff741000b790a3772f2961a52/src/policy/rbf.cpp#L37

Reply:
It wasn't the goal to document `RBFTransactionState`, but since you asked for it, I've done that